contracts/prompt-wars/src/contract.rs

```diff
@@ -239,13 +239,18 @@ impl Market {
             env::panic_str("ERR_SELF_DESTRUCT_WINDOW_NOT_EXPIRED");
         }
 
-        if !self.get_outcome_ids().is_empty()
-            && !self.fees.claimed_at.is_some()
-            && self.collateral_token.fee_balance > 0
-        {
+        if !self.get_outcome_ids().is_empty() {
+            env::panic_str("ERR_NO_OUTCOME_IDS");
+        }
+
+        if !self.fees.claimed_at.is_some() {
             env::panic_str("ERR_SELF_DESTRUCT_FEES_UNCLAIMED");
         }
 
+        if self.collateral_token.fee_balance > 0 {
+            env::panic_str("ERR_COLLATERAL_TOKEN_FEE_BALANCE_GREATER_THAN_ZERO");
+        }
+
```

Comment:
@netpoe done. is this fine?

Reply:
Let me try it out. I'll let you know. Thx.